velox/exec/RowContainer.cpp

```diff
@@ -585,8 +600,8 @@ void RowContainer::extractSerializedRows(
     size_t offset = 0;
 
     // Copy nulls.
```

Comment:
```
// Copy null and initialized flags.
```

Reply:
It copies nulls, initialized flags, the probed flag, etc.

So I just changed it to
// Copy nulls and other flags.